crates/db_schema/src/impls/post.rs

```diff
@@ -254,6 +262,7 @@ impl Post {
     post::table
       .filter(post::ap_id.eq(object_id))
       .filter(post::scheduled_publish_time_at.is_null())
+      .select(Self::as_select())
```

Comment:
Why are you adding these explicit select calls?

Reply:
Its diesel's preferred way to select from the main table, its harmless.